pkg/packagekit/assets/distribution.dist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<installer-gui-script minSpecVersion="2">
+	<title>{{.Title}}</title>
+	<pkg-ref id="com.{{.Identifier}}.launcher">
```

Comment:
I think the `com.{{.Identifier}}.launcher` isn't going to be what we want, and we should treat this the same at `.Title` (eg: put more logic in go)

I think that our builds, should probably come out as `com.kolide.agent` and I have no idea what other people would need.

Reply:
I did it this way because this is how our build currently generates the ID. Would we want it to change now?

Reply:
Hrm. I guess I'm not sure. What is this id anyhow?

Our current production one is `kolide-k2`, and `com.kolide-k2.launcher` is kinda weird, and probably not correct for our bundle id. But I'm less sure what the pkg id is used for.

Reply:
I have no idea. 

As you said, this is how it's set to day, maybe better to leave it unchanged. And assume we'll revisit in the future